content/en/blog/general/apply-vap-using-cli/index.md

```diff
@@ -0,0 +1,222 @@
+    - apiGroups:   ["apps"]
+      apiVersions: ["v1"]
+      operations:  ["CREATE", "UPDATE"]
+      resources:   ["deployments"]
+  validations:
+    - expression: "!has(object.spec.template.spec.volumes) || object.spec.template.spec.volumes.all(volume, !has(volume.hostPath))"
+      message: "HostPath volumes are forbidden. The field spec.template.spec.volumes[*].hostPath must be unset."
+EOF
+```
+
+Then create the two deployments, one of which violates the policy:
+```sh
+cat << EOF > ./test-dir/deployments.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: deployment-pass
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+      - name: nginx-server
+        image: nginx
+        volumeMounts:
+          - name: temp
+            mountPath: /scratch
+      volumes:
+      - name: temp
+        emptyDir: {}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: deployment-fail
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+      - name: nginx-server
+        image: nginx
+        volumeMounts:
+          - name: udev
+            mountPath: /data
+      volumes:
+      - name: udev
+        hostPath:
+          path: /etc/udev
+EOF
+```
+
+The tests are written in a file name `kyverno-test.yaml` so you will create two tests, one for each deployment and test them aganist the policy.
+```sh
+cat << EOF > ./test-dir/kyverno-test.yaml
+name: disallow-host-path-test
+policies:
+  - disallow-host-path.yaml
+resources:
+  - deployments.yaml
+results:
+  - policy: disallow-host-path
+    resource: deployment-pass
+    isValidatingAdmissionPolicy: true
+    kind: Deployment
+    result: pass
+  - policy: disallow-host-path
+    resource: deployment-fail
+    isValidatingAdmissionPolicy: true
+    kind: Deployment
+    result: fail
+EOF
+```
+Now, we’re ready to test the two deployments aganist validating admission policy.
+```sh
+kyverno test ./test-dir
+
+Executing disallow-host-path-test...
+
+│────│────────────────────│──────│────────────────────────────│────────│────────│
+│ ID │ POLICY             │ RULE │ RESOURCE                   │ RESULT │ REASON │
+│────│────────────────────│──────│────────────────────────────│────────│────────│
+│  1 │ disallow-host-path │      │ Deployment/deployment-pass │ Pass   │ Ok     │
+│  2 │ disallow-host-path │      │ Deployment/deployment-fail │ Pass   │ Ok     │
+│────│────────────────────│──────│────────────────────────────│────────│────────│
+
+Test Summary: 2 tests passed and 0 tests failed
+```
+As expected, the two tests passed because the actual result of each test matches the desired result as defined in the test manifest.
+
+## Conclusion
+This blog post explains how to apply validating admission policies to resources using Kyverno CLI.
```

Comment:
We need a bit more of a conclusion here. This is a new capability. Why is it useful?